egs/librispeech/asr/simple_v1/bpe_run.sh

```diff
@@ -25,9 +29,80 @@ if [ ! -f exp/data/cuts_test-clean.json.gz ]; then
 fi
 
 if [ $stage -le 1 ]; then
-  export CUDA_VISIBLE_DEVICES=3
+  local/download_lm.sh "openslr.org/resources/11" data/local/lm
+fi
+
+if [ $stage -le 2 ]; then
+  dir=data/lang_bpe2
+  mkdir -p $dir
+  token_file=./data/en_token_list/bpe_unigram5000/tokens.txt
```

Comment:
Where does this come from (data/en_token_list/bpe_unigram5000/tokens.txt)?

Reply:
Currently they are downloaded from the [snowfall_model_zoo](https://huggingface.co/GuoLiyong/snowfall_bpe_model) together with neural net models.
Originally, They are  trained by sentencepiece tokenizer which is also used by Espnet and #215. 
To make it easier to be reviewed,  this pr is mainly about decoding part and tokenizer training part will be summited with the model training part #219.